### PR TITLE
Niloofar | Refactoring translations (localize / Localize) | FE Core Sprints [30 Jan - 19 Feb] [parent]

### PR DESCRIPTION
**Pull request by niloofar-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: crowdin/messages.json [hunk 1]
```diff
@@ -2185,10 +2167,11 @@
   "-370313455": "The Deriv MT5 Standard account offers new and experienced traders high leverage and variable spreads for maximum flexibility.<br />The Deriv MT5 Advanced account is a 100% A Book account where your trades are passed straight through to the market, giving you direct access to forex liquidity providers.<br />The Deriv MT5 Synthetic Indices account allows you to trade contracts for difference (CFDs) on synthetic indices that mimic real-world movements. It is available for trading 24/7 and audited for fairness by an independent third party.",
   "-1508841947": "DTrader allows you to trade more than 50 assets in the form of digitals, multiplier, and lookback options.<br />Deriv MT5 is a multi-asset trading platform that you can use to trade spot forex and contracts for difference (CFDs) with leverage.",
   "-2073101307": "<0>View fewer questions</0>",
+  "-369631423": "Frequently asked questions - ${untranslate_category}",
```
Comment (mohsen-deriv) [inline crowdin/messages.json:2170]: this shouldn't be like this, please use the interpolation feature of i18next, thanks :pray:

File: src/pages/help-centre/components/_questions-template.tsx [hunk 2]
```diff
@@ -32,9 +32,9 @@ const QuestionsTemplate = ({ data }: TQuestionsTemplate) => {
         <Layout>
             <SEO
                 title={localize(
-                    `Help centre | Frequently asked questions | ${untranslate_category} | Deriv`,
+                    `_t_Help centre | Frequently asked questions | ${untranslate_category} | Deriv_t_`,
                 )}
-                description={localize(`Frequently asked questions - ${untranslate_category}`)}
+                description={localize(`_t_Frequently asked questions - ${untranslate_category}_t_`)}
```
Comment (mohsen-deriv) [inline src/pages/help-centre/components/_questions-template.tsx:37]:
```suggestion
                description={localize(`_t_Frequently asked questions - {{untranslate_category}}_t_`, ${untranslate_category})}
```

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

**Review by mohsen-deriv — APPROVED** [review]

File: src/pages/who-we-are/_data.tsx [hunk 3]
```diff
@@ -1,227 +1,212 @@
-import React, { ReactElement } from 'react'
-import { Localize } from 'components/localization'
 import Shield from 'images/svg/who-we-are/shield.svg'
 import Star from 'images/svg/who-we-are/star.svg'
 import People from 'images/svg/who-we-are/people.svg'
 import Hands from 'images/svg/who-we-are/hands.svg'
 import { all_offices, all_countries } from 'pages/careers/_model/_locations/_locations'
+import { TString } from 'types/generics'
+
+const number_of_offices = all_offices.length.toString()
```
Comment (niloofar-deriv) [inline src/pages/who-we-are/_data.tsx:8]: @mohsen-deriv mohsen could u pls check this too, is that ok to use it like this?

Comment (mohsen-deriv) [reply src/pages/who-we-are/_data.tsx:8]: yes it is, for now let's use this approach. thanks :pray:

Comment (niloofar-deriv) [reply src/pages/who-we-are/_data.tsx:8]: Thanks for checking :woman_dancing:

**Pull request closed without merging**